index.node.js.tpl

```diff
@@ -0,0 +1,5 @@
+const fetch = require('node-fetch')
+const { URL, URLSearchParams } = require('url')
+
+{{body}}
```

Comment:
I wonder if there's really no other way to include js code, templating looks a bit like overkill

Reply:
@dy seemed more straight forward then to include something like webpack. How else could this be done?

Reply:
@zeke what are your thoughts on this?

Reply:
I think it's fine. Easy to reason about. Lightweight. No new deps. 👍🏼